src/Microsoft.DotNet.Interactive.Tests/LanguageKernelFormattingTests.cs

```diff
@@ -145,7 +146,69 @@ await kernel.FindKernelByName("csharp").As<CSharpKernel>()
     [Theory]
     [InlineData(Language.CSharp)]
     [InlineData(Language.FSharp)]
-    public async Task Display_helper_can_be_called_without_specifying_class_name(Language language)
+    public async Task DisplayTable_produces_tabular_HTML_output_for_IEnumerable_T(Language language)
```

Comment:
Should there be another test to validate what happens when the collection contains instances with heterogeneous types?